src/adr/0036-against-use-of-conditional-statements-in-guice-modules.md

```diff
@@ -36,17 +36,19 @@ Here is the list of products we provide:
  - In-Memory: A memory based James server, mainly for testing purposes
  - Distributed James: A scalable James server, storing data in various data stores. Cassandra is used for metadata, 
  ElasticSearch for search, RabbitMQ for messaging, and ObjectStorage for blobs.
- - Cassandra: An implementation step toward Distributed James. It does not include messaging and ObjectStorage and should not be run in a cluster way but is still relevant for good performance.
+ - Cassandra: An implementation step toward Distributed James. It does not include messaging and ObjectStorage and 
+ should not be run in a cluster way but is still relevant for good performance. This product is about to be deprecated
```

Comment:
I still don't agree to include "This product is about to be deprecated" here, it's a useless out-of-context addition.